### PR TITLE
Fix for ios resize + universal storyboard

### LINKED ISSUE #241: iOS Playground Crashing With MTLRenderPassDescriptor texture and resolveTexture size mismatch
I know this is a heavy work in progress but I'm trying to get the iOS playground working but can't seem to get past this crash after successfully building in Xcode. 

```
_MTLValidateResolveTexture:328: failed assertion `MTLRenderPassDescriptor texture and resolveTexture size mismatch.'
```

With the crash taking me to line 167-169 at `BabylonNative/Dependencies/bgfx/bgfx/Header Files/renderer_mtl.h`

```
// Creating Command Encoders
  id<MTLRenderCommandEncoder>renderCommandEncoderWithDescriptor(MTLRenderPassDescriptor* _renderPassDescriptor) {
     return [m_obj renderCommandEncoderWithDescriptor:_renderPassDescriptor];
}
```
Have tried deleting then re-cloning and following build instructions a couple times with no luck, and was able to get the MacOS build target to work with no problems. Also tried building on simulator and real device with same outcome. 

I'm on MacOS 10.15.2 and Xcode 11.4.1. Any help would be greatly appreciated! 

Edit: Added the cmake output below which I imagine would be helpful 

```
-- Merging integrated CMake 3.14+ iOS,tvOS,watchOS,macOS toolchain(s) with this toolchain!
-- Building with Xcode version: 11.4.1
-- Configuring iphoneos build for platform: OS64COMBINED, architecture(s): arm64;x86_64
-- Using SDK: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk for platform: OS64COMBINED
-- Disabling bitcode support by default on simulators. ENABLE_BITCODE not provided for override!
-- Hiding symbols visibility by default. ENABLE_VISIBILITY not provided!
-- Using NON-strict compiler checks by default. ENABLE_STRICT_TRY_COMPILE not provided!
-- Using C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
-- Using CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++
-- Using libtool: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool
-- Using install_name_tool: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/install_name_tool
-- Will combine built (static) artifacts into FAT lib...
-- Using a data_ptr size of 8
-- Building for minimum iphoneos version: 12 (SDK version: 13.4)
-- Version flags set to: -mios-simulator-version-min=12
-- Disabling bitcode support.
-- Disabling ARC support.
-- Hiding symbols (-fvisibility=hidden).
-- Not setting any manual command-line buildflags, since Xcode is selected as generator.
-- Merging integrated CMake 3.14+ iOS,tvOS,watchOS,macOS toolchain(s) with this toolchain!
-- Building with Xcode version: 11.4.1
-- Configuring iphoneos build for platform: OS64COMBINED, architecture(s): arm64;x86_64
-- Using SDK: iphoneos for platform: OS64COMBINED when checking compatibility
-- Using libtool: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool
-- Will combine built (static) artifacts into FAT lib...
-- Using a data_ptr size of 8
-- Building for minimum iphoneos version: 12 (SDK version: 13.4)
-- Version flags set to: -mios-simulator-version-min=12
-- Disabling bitcode support.
-- Disabling ARC support.
-- Hiding symbols (-fvisibility=hidden).
-- Not setting any manual command-line buildflags, since Xcode is selected as generator.
-- The C compiler identification is AppleClang 11.0.3.11030032
-- The CXX compiler identification is AppleClang 11.0.3.11030032
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Git: /usr/bin/git (found version "2.24.2 (Apple Git-127)")
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Finding Git version for SPIRV-Cross.
-- Git hash: 3e53273c
-- The Swift compiler identification is Apple 5.2.2
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/kanethomas/dev-projects/BabylonNative/Build
```

### PR DESCRIPTION
**Pull request by CedricGuillemet**
closes #241 
Partial fix, I still get issues with wrong dimensions reported. But the crash is fixed for now. I also changed the storyboard to work with any screen resolutions.

### CODE REVIEW

File: Apps/Playground/iOS/Base.lproj/LaunchScreen.storyboard [hunk 1]
```diff
@@ -11,9 +13,9 @@
             <objects>
                 <viewController id="01J-lp-oVM" sceneMemberID="viewController">
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
```
Comment (bghgary) [inline Apps/Playground/iOS/Base.lproj/LaunchScreen.storyboard:16]: arbitrary numbers?

File: Apps/Playground/iOS/ViewController.swift [hunk 2]
```diff
@@ -26,22 +30,24 @@ class ViewController: UIViewController {
...
+        }
         if appDelegate != nil {
-            let screenBounds = UIScreen.main.bounds
-            let width = screenBounds.size.width
-            let height = screenBounds.size.height
+            let width = view.bounds.size.width
+            let height = view.bounds.size.height
 
-            appDelegate!._bridge!.resize(Int32(width), height: Int32(height))
+            appDelegate!._bridge!.resize(Int32(width + 1), height: Int32(height + 1))
```
Comment (bghgary) [inline Apps/Playground/iOS/ViewController.swift:50]: i'm assuming there is a reason for the `+ 1`. maybe add a comment?

File: Plugins/NativeEngine/Source/NativeEngine.h [hunk 3]
```diff
@@ -88,6 +88,7 @@ namespace Babylon
             bgfx::setViewClear(m_viewId, m_flags, Color(), m_depth, m_stencil);
             // discard any previous set state
             bgfx::discard();
+            bgfx::touch(0);
```
Comment (bghgary) [inline Plugins/NativeEngine/Source/NativeEngine.h:91]: same here. is there a reasoning for this?

File: Apps/Playground/iOS/Base.lproj/LaunchScreen.storyboard (hunk 1 above)
Comment (CedricGuillemet) [reply Apps/Playground/iOS/Base.lproj/LaunchScreen.storyboard:16]: Interface builder changed values when I tested with other resolution.

File: Apps/Playground/iOS/ViewController.swift (hunk 2 above)
Comment (CedricGuillemet) [reply Apps/Playground/iOS/ViewController.swift:50]: I'll spend more time on this one. I don't really understand some of the results I get.

File: Plugins/NativeEngine/Source/NativeEngine.h (hunk 3 above)
Comment (CedricGuillemet) [reply Plugins/NativeEngine/Source/NativeEngine.h:91]: That's the fix! I've updated this issue https://github.com/BabylonJS/BabylonNative/issues/118

Comment (bghgary) [reply Plugins/NativeEngine/Source/NativeEngine.h:91]: Sure, but why is it necessary?

Comment (CedricGuillemet) [reply Plugins/NativeEngine/Source/NativeEngine.h]: Removed.

File: Apps/Playground/iOS/ViewController.swift (hunk 2 above)
Comment (CedricGuillemet) [reply Apps/Playground/iOS/ViewController.swift]: Confirmed to be working on device & sim.

**Pull request merged**